.github/workflows/mastodon-ruby-tests.yml

```diff
@@ -0,0 +1,86 @@
+name: mastodon-ruby-tests
+on:
+  workflow_dispatch:
```

Comment:
Add a schedule run to run once a day
    schedule:
      - cron: '0 6 * * *' # run at 6 AM UTC

Reply:
In last week's meeting we agreed to run this only on demand cc @romange

Reply:
Sorry I do not remember we discussed this
why only on demand @romange ?

Reply:
No I said we should do it via crontab and not for every PR and you said for now you added "on demand".
We should of course have crontab - can be now as well